From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaroudn for ath9k ath_node.ackto type change
Patch-mainline: Never, SLE15 only
References: bsc#1051510

The patch patches.drivers/ath9k-dynack-make-ewma-estimation-faster.patch
changed the struct ath_node ackto field type.  Applied a typical
workaround with ifdef.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/net/wireless/ath/ath9k/ath9k.h |    4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/net/wireless/ath/ath9k/ath9k.h
+++ b/drivers/net/wireless/ath/ath9k/ath9k.h
@@ -272,7 +272,11 @@ struct ath_node {
 #endif
 	u8 key_idx[4];
 
+#ifdef __GENKSYMS__
+	u32 ackto;
+#else
 	int ackto;
+#endif
 	struct list_head list;
 };
 
